backend/app/assets/javascripts/spree/backend/stock_transfer.js.coffee

```diff
@@ -1,196 +0,0 @@
-$ ->
```

Comment:
This was deleted because I believe it is dead code. We have no `#transfer_variant_template`, no `#transfer-variants-table` etc anywhere in our codebase.